osu.Game/Screens/Ranking/Statistics/HitEventTimingDistributionGraph.cs

```diff
@@ -295,6 +330,15 @@ protected override void LoadComplete()
                     boxOriginals[i].ResizeHeightTo(heightForValue(values[i].Value), duration, Easing.OutQuint);
                     offsetValue -= values[i].Value;
                 }
+
+                basalHeight.BindValueChanged(_ => draw());
+            }
+
+            protected override void Update()
+            {
+                base.Update();
+
+                basalHeight.Set(calculateBasalHeight());
```

Comment:
I've never seen `Set()` be used before in our code. It's only supposed to be used when the incoming argument type differs from the bindable type (`float`). Since this is not the case, please change all these to use `Value = ` instead.